cw-orch-mock/src/core.rs

```diff
@@ -489,7 +484,7 @@ mod test {
         let contract_id = "my_contract";
         let code_id = 1u64;
         let address = &Addr::unchecked(BALANCE_ADDR);
-        let mut mock_state = Rc::new(RefCell::new(MockState::new()));
+        let mut mock_state = RcState(Rc::new(RefCell::new(MockState::new())));
```

Comment:
We could have a nice `new` fn for `RcState` so that callers can do `RcState::new(MockState::new())`